tree/ntuple/v7/inc/ROOT/RNTupleDescriptor.hxx

```diff
@@ -137,6 +137,9 @@ public:
    std::uint32_t GetColumnCardinality() const { return fColumnCardinality; }
    std::optional<std::uint32_t> GetTypeChecksum() const { return fTypeChecksum; }
    bool IsProjectedField() const { return fProjectionSourceId != kInvalidDescriptorId; }
+   /// Tells if the field describes a user-defined class with a dictionary.
```

Comment:
Even-though the dictionary is necessary in those cases, is that the real discriminant?  From what I can tell this is marking the difference between the "natively" supported classes.   (technically any of the STL collection have a dictionary - in particular to get the CollectionProxy).